### ISSUE TITLE
Test result not consistent

### ISSUE THREAD

**Issue by Davis980520** [yongqyu/gc-mc-pytorch#7]
Hi! Thank you for sharing the code.
However, when I run the test part of the code each time, the code produces different outputs. (I did not change any part of the code, only printing out m_hat.) model.eval() has disabled dropout, so I can't see which part of the code may cause this randomness.

**Comment by yongqyu** [comment]
Hi, Davis.
I can't understand the situation either. 
Please check whether the same symptom occurs, after fixing the random seed.
Thanks.

**Comment by Davis980520** [comment]
Thank you for your reply!
I haven't tried fix all random seeds(both numpy and torch), but it still produces different results.
